library/kani_macros/src/lib.rs

```diff
@@ -49,3 +50,26 @@ pub fn proof(_attr: TokenStream, item: TokenStream) -> TokenStream {
     //     $item
     // )
 }
+
+#[cfg(not(kani))]
+#[proc_macro_attribute]
+pub fn unwind(_attr: TokenStream, _item: TokenStream) -> TokenStream {
+    // When the config is not kani, we should leave the function alone
+    _item
```

Comment:
Does it issue a warning or anything about the _? (That usually indicated unused, so maybe this should just be `item` not `_item`)